LBD2024G06TP1Script.sql

```diff
@@ -56,45 +58,67 @@ CREATE TABLE IF NOT EXISTS `LBD2024G06AGROSA`.`EMPLEADOS` (
   `cuil` INT(11) UNSIGNED NOT NULL,
   `nombres` VARCHAR(30) NOT NULL,
   `apellidos` VARCHAR(30) NOT NULL,
-  `estado` CHAR(1) NOT NULL,
+  `estado` CHAR(1) NOT NULL DEFAULT 'A',
   PRIMARY KEY (`idEMPLEADO`),
   UNIQUE INDEX `idEMPLEADOS_UNIQUE` (`idEMPLEADO` ASC) VISIBLE)
 ENGINE = InnoDB
 DEFAULT CHARACTER SET = utf8;
 
+-- Added table DUEÑOS
+
+CREATE TABLE IF NOT EXISTS `LBD2024G06AGROSA`.`DUEÑOS` (
```

Comment:
Please don't use ñ for table names. Maybe we can use a table called PROPIETARIOS